deploy/services/script/service_setup.py

```diff
@@ -0,0 +1,186 @@
+import platform 
+import os
+from pathlib import Path
+import os
+import platform
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+from typing import Tuple
+from dotenv import load_dotenv
```

Comment:
We need requirements.txt if this is a pip package. Please place this in `script/` directory and update the README